TODO.md

```
@@ -0,0 +1,266 @@
+
+---
+
+## Deviations from Spec
+
+### 1. Token Base64 Encoding (RSA3, RSA4a, RSA4)
+
+**Issue:** The spec expects Bearer tokens to be sent raw, but ably-go base64 encodes them.
+
+| Expected (per spec) | Actual (ably-go) |
+|---------------------|------------------|
+| `Bearer my-token-string` | `Bearer bXktdG9rZW4tc3RyaW5n` |
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)